v1/charm-dev.yaml

```diff
@@ -110,9 +121,11 @@ instances:
           # The dns addon will restart the api server so you may see a blip in the availability
           # Separating addons to avoid errors such as
           # dial tcp 127.0.0.1:16443: connect: connection refused
-          microk8s.enable dns rbac
+          microk8s.enable dns
           microk8s.kubectl rollout status deployments/coredns -n kube-system -w --timeout=600s
 
+          microk8s.enable rbac
+
```

Comment:
Just curious why the change to separate `micork8s.enable rbac` line?

Reply:
Enabling multiples is deprecated, as different addons may interfere with each other.

Reply:
Ok, makes sense.  Thanks!